src/status_im/contexts/profile/contact/contact_review/view.cljs

```diff
@@ -0,0 +1,47 @@
+(ns status-im.contexts.profile.contact.contact-review.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.contexts.profile.contact.contact-review.style :as style]
+            [status-im.contexts.profile.utils :as profile.utils]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [{:keys [public-key customization-color]
+         :as   profile}                      (rf/sub [:contacts/current-contact])
+        {contact-request-id :id
+         :as                contact-request} (rf/sub [:activity-center/contact-request-from-contact-id
+                                                      public-key])
+        ;; TODO: remove :blue when #18733 merged.
```

Comment:
According to guidelines, we have a format for TODO comments, but generally we avoid them and prefer to use GH issues and add any necessary details there. See https://github.com/status-im/status-mobile/blob/62f68ff44b10173946c8bf5919d3209f8836c22d/doc/new-guidelines.md#using-todos-comments